Add xsettingsd package

clefru opened the pull request:
First contribution. Please let me know if there's something to improve.

clefru committed:
Add xsettingsd.

lethalman commented on pkgs/tools/X11/xsettingsd/default.nix:
```diff
@@ -0,0 +1,40 @@
+
+  buildInputs = [ libX11 scons pkgconfig ];
+  buildPhase = ''
+    runHook preBuild
```
Doesn't make much sense to `runHook preBuild` here, you can remove it.

lethalman commented on pkgs/tools/X11/xsettingsd/default.nix:
```diff
@@ -0,0 +1,40 @@
+  '';
+  
+  installPhase = ''
+    runHook preInstall
```
Same here.

lethalman commented on pkgs/tools/X11/xsettingsd/default.nix:
```diff
@@ -0,0 +1,40 @@
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Provides settings to X11 applications via the XSETTINGS specification.";
```
A nitpick, can you please remove that trailing dot in the description? :)

lethalman commented:
Welcome. I've commented a couple of nitpicks just to line out some of nixpkgs conventions, otherwise the package looks quite good :)

jagajaga commented on pkgs/tools/X11/xsettingsd/default.nix:
```diff
@@ -0,0 +1,40 @@
+  name = "xsettingsd-${version}";
+  version = "0.0.1";
+
+  src = fetchgit {
```
Better use `fetchFromGithub`

clefru commented:
Thanks for the input. I addressed all remarks.

My package builds successfully locally, however Travis seems to not like it. From its logs it doesn't seems that my package is the culprit but rather something internal with Travis isn't right. Should I be worried?

lethalman commented:
Built locally, thanks.
